docs/_posts/2019-05-03-baremetal-installation-guide.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: page
+title: Installing MISO on baremetal
+category: adm
+date: 2016-01-11 13:51:46
+order: 2
+---
+
+This installation guide is intended to be used if you cannot use Docker and
+Docker compose, and is not trivial to set up. We recommend using Docker compose
+if possible by following the
+[Docker compose installation guide](compose-installation-guide).
+
+MISO requires some configuration directly in the source code. While we plan to
+change this over time, running an instance of MISO will require building and
+deploying a fork of the code base with customisations.
```

Comment:
I believe that this is no longer accurate, as the `miso.properties` file is read from Tomcat instead of from within the WAR.